src/sidebar/panel.html

```diff
@@ -42,13 +45,24 @@
 
     <div id="footer-buttons">
       <button id="enable-sync"></button>
-      <a target="_blank" id="give-feedback"></a>
+			<div class="wrapper">
+				<button id="demo-menu-top-right"
```

Comment:
Indenting here looks a bit funny too.

Also, should the button's ID be a bit more... informative (or removed if we're not using it)?